mmv1/products/vertexai/api.yaml

```diff
@@ -103,6 +103,238 @@ objects:
+                description: |
+                  Immutable. The specification of a single machine used by the prediction.
+                properties:
+                  - !ruby/object:Api::Type::String
+                    name: 'machineType'
+                    description: |
+                      Immutable. The type of the machine.  
+                      See the [list of machine types supported for prediction](https://cloud.google.com/vertex-ai/docs/predictions/configure-compute#machine-types)  
+                      See the [list of machine types supported for custom training](https://cloud.google.com/vertex-ai/docs/training/configure-compute#machine-types).  
+                      For `DeployedModel` this field is optional, and the default value is `n1-standard-2`. For `BatchPredictionJob` or as part of `WorkerPoolSpec` this field is required.  
+                  - !ruby/object:Api::Type::Enum
+                    name: 'acceleratorType'
+                    description: |
+                      Immutable. The type of accelerator(s) that may be attached to the machine as per acceleratorCount.
+                    values:
+                          - ACCELERATOR_TYPE_UNSPECIFIED
+                          - NVIDIA_TESLA_K80
+                          - NVIDIA_TESLA_P100
+                          - NVIDIA_TESLA_V100
+                          - NVIDIA_TESLA_P4
+                          - NVIDIA_TESLA_T4
+                          - NVIDIA_TESLA_A100
+                  - !ruby/object:Api::Type::Integer
+                    name: 'acceleratorCount'
+                    description: The number of accelerators to attach to the machine.
+              - !ruby/object:Api::Type::Integer
+                name: 'minReplicaCount'
+                description: 
+                  Immutable. The minimum number of machine replicas this DeployedModel will be always deployed on. This value must be greater than or equal to 1.  
+                  If traffic against the DeployedModel increases, it may dynamically be deployed onto more replicas, and as traffic decreases, some of these extra replicas may be freed.
+                required: true
+              - !ruby/object:Api::Type::Integer
+                name: 'maxReplicaCount'
+                description: 
+                  Immutable. The maximum number of replicas this DeployedModel may be deployed on when the traffic against it increases. If the requested value is too large, the deployment will error, but if deployment succeeds then the ability to scale the model to that many replicas is guaranteed (barring service outages). If traffic against the DeployedModel increases beyond what its replicas at maximum may handle, a portion of the traffic will be dropped.  
+                  If this value is not provided, will use `minReplicaCount` as the default value.
+                input: true
+              #   TODO: autoscalingMetricSpecs: How to add an array of nested objects? 
```

Comment:
I don't remember the exact invocation offhand, but there are examples. It's a messy resource, but Compute BackendService's `backends` field comes to mind.